scripts/rollup/bundles.js

```diff
@@ -369,6 +369,25 @@ const bundles = [
   },
 ];
 
+// Based on deep-freeze by substack (public domain)
+function deepFreeze(o) {
+  Object.freeze(o);
+  Object.getOwnPropertyNames(o).forEach(function(prop) {
+    if (
+      o.hasOwnProperty(prop) &&
+      o[prop] !== null &&
+      (typeof o[prop] === 'object' || typeof o[prop] === 'function') &&
```

Comment:
function why?

Reply:
Oh I just copy pasted. I didn't actually check the semantics, assumed it's there for a reason 😛

Reply:
I guess putting things onto functions is also not nice?